execute/exectypes/commit_data.go

```diff
@@ -25,8 +25,6 @@ type CommitData struct {
 	// ExecutedMessages are the messages in this report that have already been executed.
 	ExecutedMessages []cciptypes.SeqNum `json:"executedMessages"`
 
-	// The following values are cached for validation algorithms, serialization is not required for consensus.
-
 	// TokenData for each message.
-	TokenData [][][]byte `json:"-"`
+	MessageTokenData []MessageTokenData `json:"messageTokenData"`
```

Comment:
Making sure that `type CommitData struct {` are not the data expected to be in a commit report.

Because e.g. this comment indicates the opposite.
```
// CommitObservations contain the commit plugin report data organized by the source chain selector.
type CommitObservations map[cciptypes.ChainSelector][]CommitData
```

Reply:
No, it's not, it's just the aggregate for all the information necessary to build an exec report. Maybe naming is not perfect, but I don't want to change that as part of my PR.